test/bin/test-citgm-all.js

```diff
@@ -3,9 +3,9 @@
 var path = require('path');
 
 var test = require('tap').test;
-var spawn = require('../lib/spawn');
+var spawn = require('../../lib/spawn');
 
-var citgmAllPath = path.resolve(__dirname, '..', 'bin', 'citgm-all');
+var citgmAllPath = path.resolve(__dirname, '..', '..', 'bin', 'citgm-all.js');
```

Comment:
Same as `require.resolve('../../bin/citgm-all');`, unless there's a reason to avoid it.